src/plugins/auth/index.js

```diff
@@ -43,29 +43,31 @@ export const auth = {
             }
           }
 
-          const { preferred_username: preferredUsername, groups = [] } = user
+          const { oid, groups = [] } = user
 
-          if (!preferredUsername) {
-            logger.error('Authentication error: Missing preferred_username')
+          if (!oid || typeof oid !== 'string') {
```

Comment:
Mind if we update **types.js** on this repo too?

```patch
  /**
-  * @typedef {IdTokenClaims & { groups?: string[] }} UserProfile
+  * @typedef {IdTokenClaims & { groups?: string[], oid?: string }} UserProfile
   */
```



```suggestion
          if (!oid) {
```

Reply:
You can tell it's been a long week when I'm missing the obvious stuff 🤦

Thanks